src/main/java/kr/codesqaud/cafe/domain/Article.java

```diff
@@ -0,0 +1,53 @@
+package kr.codesqaud.cafe.domain;
+
+public class Article {
+    private long index;
+    private String writer;
+    private String title;
+    private String contents;
+    private String time;
```

Comment:
```suggestion
    private LocalDateTime createdTime;
```
1) 데이터는 원본을 저장할 것 
2) 자바의 날짜 표현에는 어떤 것들이 있는지 확인할 것